dev/Dockerfile-dev

```diff
@@ -18,6 +18,7 @@ COPY Makefile Makefile
 COPY setup.cfg .
 
 RUN chmod 777 -R .
+RUN chmod 777 /root
```

Comment:
Why do you need this?

Reply:
This helps VS Code to attach and install it's the remote debugger in the container. Honor is to Finn;-). Next step could be to debug the code really in VSCode instead of using pdb

Reply:
Ah, ok